<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdgBancos</class>
 <widget class="QWidget" name="wdgBancos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="../images/xulpymoney.qrc">
    <normaloff>:/xulpymoney/bank.png</normaloff>:/xulpymoney/bank.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="lbl">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Banks list</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="chkInactivas">
           <property name="text">
            <string>Show inactive banks</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="myQTableWidget" name="tblEB">
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Banks</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Active</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Balance</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Accounts</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="myQTableWidget" name="tblCuentas">
            <property name="contextMenuPolicy">
             <enum>Qt::CustomContextMenu</enum>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Account</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Active</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Balance</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Investments</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="myQTableWidget" name="tblInversiones">
            <property name="contextMenuPolicy">
             <enum>Qt::CustomContextMenu</enum>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Investment</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Active</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Balance</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionInversionEstudio">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/report2.png</normaloff>:/xulpymoney/report2.png</iconset>
   </property>
   <property name="text">
    <string>Estudio de la Inversión</string>
   </property>
  </action>
  <action name="actionCuentaEstudio">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/report2.png</normaloff>:/xulpymoney/report2.png</iconset>
   </property>
   <property name="text">
    <string>Estudio de cuentas</string>
   </property>
   <property name="toolTip">
    <string>Estudio de cuentas</string>
   </property>
  </action>
  <action name="actionBancoNuevo">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/add.png</normaloff>:/xulpymoney/add.png</iconset>
   </property>
   <property name="text">
    <string>Nuevo Banco</string>
   </property>
   <property name="toolTip">
    <string>Nuevo Banco</string>
   </property>
  </action>
  <action name="actionBancoBorrar">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/button_cancel.png</normaloff>:/xulpymoney/button_cancel.png</iconset>
   </property>
   <property name="text">
    <string>Borrar Banco</string>
   </property>
   <property name="toolTip">
    <string>Borrar Banco</string>
   </property>
  </action>
  <action name="actionBancoModificar">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/editar.png</normaloff>:/xulpymoney/editar.png</iconset>
   </property>
   <property name="text">
    <string>Modificar Banco</string>
   </property>
   <property name="toolTip">
    <string>Modificar Banco</string>
   </property>
  </action>
  <action name="actionActiva">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>¿Está activo?</string>
   </property>
   <property name="toolTip">
    <string>¿Está activo?</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myQTableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">myqtablewidget</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images/xulpymoney.qrc"/>
 </resources>
 <connections/>
</ui>
